packages/lib-core/src/types/core.ts

```diff
@@ -0,0 +1,18 @@
+export type K8sVerb =
+  | 'create'
+  | 'get'
+  | 'list'
+  | 'update'
+  | 'patch'
+  | 'delete'
+  | 'deletecollection'
+  | 'watch';
+
+export type AccessReviewResourceAttributes = {
+  group?: string;
+  resource?: string;
+  subresource?: string;
+  verb?: K8sVerb;
+  name?: string;
+  namespace?: string;
+};
```

Comment:
I feel this is an "OpenShift Console"-esk of a type and not an accurate type. We had a serious problem (less so now) about types in OpenShift Console being fully optional... making our code a mess of if-statements.

May need a bit of investigation 🤔

Reply:
Ping, if we don't need `AccessReviewResourceAttributes` let's remove it for now.

Reply:
It's used in `actions.ts` I'll remove the optional prop that is typed by this.